pythonforandroid/recipe.py

```diff
@@ -1266,8 +1306,11 @@ def install_wheel(self, arch, built_wheels):
 
     def build_arch(self, arch):
         self.install_hostpython_prerequisites(
-            packages=["build[virtualenv]", "pip"] + self.hostpython_prerequisites
+            packages=["build[virtualenv]", "pip", "setuptools"] + self.hostpython_prerequisites
         )
+        python_bin_dir = join(self._host_recipe.site_dir, "bin")
+        self.patch_shebangs(python_bin_dir, self.real_hostpython_location)
```

Comment:
Do you know why it performs build wheel step in the next lines?
I am getting an error building setuptools here, but I also can not understand why this step is required, as far as I know it should be enough just pip install, right?

Reply:
Even worst, I dont get why it tries to build wheel from the build dir and not from the source/install dir?
setuptools does not put any pyproject or setup.py in the build dir.

Reply:
Should be fixed now.